src/cms/api/save.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Save the finished page and its assets to a zip file... 

..uses tar cli options to replace the editable page with the non-editable version, removes unneeded stuff (build/cms templates, etc)..